CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 ### Changed
 
 - The content of the field `timestamp` is migrated to `creationdate`. In case one configured "udpate timestampe", it is migrated to `modificationdate`. [koppor#130](https://github.com/koppor/jabref/issues/130)
+- The export to MS Office XML now uses the month name for the field `MonthAcessed` instead of the two digit number [#7354](https://github.com/JabRef/jabref/issues/7354)
```

Comment:
I don't think a changelog entry is necessary as this is no change relative to the last release.

Reply:
Sure it is a change relative to the new release. This applies to all MontAccessed fields, not just for urldate